File: Beecrowd/1. Beginner/1072/1072.cpp
#include <iostream>
using namespace std;
int main () {
    int n, a, in_count=0, out_count=0;

    cin >> n;

    for(int i=0; i<n; i++) {
        cin >> a;
        if(a >= 10 && a <= 20) {
            in_count++;
        }
        else {
            out_count++;
        }
    }

    cout << in_count << " in" << endl;
    cout << out_count << " out" << endl;

return 0;
}

File: Beecrowd/1. Beginner/1072/1072.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int a, inCount=0, outCount=0;
        int n = input.nextInt();

        for(int i=0; i<n; i++) {
            a = input.nextInt();
            if (a >= 10 && a <= 20){
                inCount++;
            } else {
                outCount++;
            }
        }

        System.out.println(inCount + " in");
        System.out.println(outCount + " out");

        input.close();
    }
}
